crates/xtask-unpublished/src/xtask.rs

```diff
@@ -0,0 +1,157 @@
+use cargo::core::registry::PackageRegistry;
+use cargo::core::QueryKind;
+use cargo::core::Registry;
+use cargo::core::SourceId;
+use cargo::util::command_prelude::*;
+
+pub fn cli() -> clap::Command {
+    clap::Command::new("xtask-unpublished")
```

Comment:
We should document it. Let's do it after figuring out a better look of xtask for cargo.